flybywiresim/a32nx
# fix(model): flickering displays

Fixes #7514

## Summary of Changes

Fixed flickering screens in Sim Update 10 with TAA by moving the glass forward by 1mm.

## Screenshots (if necessary)



## References





## Additional context

The issue was caused by the glass z-fighting with the screens. For some reason, this only happens when TAA is selected as the anti-aliasing method.


## Testing instructions

1. Set your anti-aliasing method to TAA
2. Ensure screens are no longer flickering


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
